connect_ext_ppr/tasks_manager.py

```diff
@@ -14,15 +17,47 @@
 from connect_ext_ppr.models.task import Task
 
 
-def validate_ppr():
+class TaskException(Exception):
+    pass
+
+
+def _send_ppr(cbc_service, file: BufferedReader):
+
+    parsed_ppr = cbc_service.parse_ppr(file)
+
+    if 'error' in parsed_ppr.keys():
+        raise TaskException(parsed_ppr.get('message'))
+
+    tracking_id = cbc_service.apply_ppr(file)
+
+    if not tracking_id:
+        raise TaskException('Some error ocurred trying to upload ppr.')
+
+    return tracking_id
+
+
+def _check_cbc_task_status(cbc_service, tracking_id):
+    task_log = cbc_service.search_task_logs_by_name(tracking_id)[0]
```

Comment:
[COD-C] Dear Agos. Can you please check that for all the HTTP requests to commerce here there is a retry mechanism? The reason is that the connection always unstable there and it will decrease number of errors

Reply:
updated!